assets/stylesheets/application.scss

```diff
@@ -2,3 +2,12 @@
 ---
 
 @import "zilker/all";
+.anchor {
```

Comment:
Not sure what the best place for these styles are, also not sure if there's a better way to make anchors work with the fixed header